integration-tests-ex/cases/cluster/Common/environment-configs/historical-for-query-error-test.env

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would be better if we put these files in a `QueryError` category to avoid cluttering `Common` with test-specific files. If we do that, then you need a query-specific `docker-compose.yaml` file. For this PR, that means copy/paste. Once the code gen PR is merged, it will mean a very short shell script.

Is this file used by multiple tests? If it is, it makes sense to go into `Common`. We might choose a name that indicates the family of tests to which it applies, such as `historical-for-query-tests.env` or some such.

Reply:
Yes, this is used by both QueryError and QueryRetry. Will rename it.